src/main/java/org/intellij/sdk/codesync/utils/DiffUtils.java

```diff
@@ -41,6 +41,7 @@ public static void writeDiffToYml(
         }
         if (isNewFile) {
             data.put("is_new_file", true);
+            data.put("added_at", CommonUtils.getCurrentDatetime());
```

Comment:
I think this should be `created_at` of the file itself. PopulateBuffer may pick new file after some time (e.g. if the file was created outside of the IDE), thoughts?

Reply:
I think we have the same behavior in VS Code as well. https://github.com/codesyncapp/codesync-vs/blob/3c5bbeeeb02c673bf9c6acb718668e066a404181/src/events/event_handler.ts#L64